apps/studio/components/interfaces/App/AppBannerWrapper.tsx

```diff
@@ -29,7 +29,7 @@ const AppBannerWrapper = ({ children }: PropsWithChildren<{}>) => {
   }, [resolvedTheme, monaco])
 
   return (
-    <div className="min-h-full flex flex-col">
+    <div className="h-screen flex flex-col">
```

Comment:
Reason behind this change? In my experience limiting the height by not using min height breaks things...

Reply:
looking now

Reply:
because currently this component wraps everything.
which it shouldn't, i've changed it in the org nav PR but for now it it's filling up the h-screen space.